eng/tools/typespec-validation/package.json

```diff
@@ -25,6 +25,6 @@
         "test": "mocha --recursive --exit"
     },
     "engines": {
-        "node": "^16.17.0 || ^18.3.0 || ^20.0.0"
+        "node": "^18.3.0 || ^20.0.0 || ^21.0.0"
```

Comment:
Just curious: why `^18.3.0` and not `^18.0.0` ?

Reply:
we were just conforming with the typespec repo for consistency

Reply:
@konrad-jamrozik: There might be some reason TypeSpec did this, for example they are using an API added in 18.3.0.    It's unlikely to matter in practice, since nobody should still be using 18.2.0 or earlier (latest is 18.19.0).